presto-main/src/main/java/com/facebook/presto/operator/scalar/DateTimeFunctions.java

```diff
@@ -107,9 +107,7 @@ public static long currentTime(ConnectorSession session)
     @SqlType(StandardTypes.TIME)
     public static long localTime(ConnectorSession session)
     {
-        // Stack value is number of milliseconds from start of the current day,
-        // but the start of the day is relative to the current time zone.
-        return getChronology(session.getTimeZoneKey()).millisOfDay().get(session.getStartTime());
+        return UTC_CHRONOLOGY.millisOfDay().get(session.getStartTime());
```

Comment:
This is not obvious that this one commit _introduces_ legacy semantics -- i.e. `localTime` implemented _new_ TIME semantics and in this commit it implements _legacy_ just to be fixed ~3 commits later.